manifest.yml

```diff
@@ -55,6 +55,13 @@ dependency_deprecation_dates:
   link: http://php.net/supported-versions.php
   match: 8.2.\d+
 dependencies:
+- name: python
```

Comment:
I don't believe automation will remove manual dependencies on version updates but please make sure if you haven't already

Reply:
No, it won't. Newrelic is a good example of a dependency that is only added manually.